### PR TITLE
2017 Spring Planning

### PR DESCRIPTION
**Pull request by kevinoliver**
A blog post about 2017 spring planning.

### CODE REVIEW

**Comment by taylorleese** [comment]
It's 2017 though.

File: source/blog/2017-03-20-spring-planning.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  twitter: kevino
+tags: Releases, Finagle, Finatra, Util, Scrooge, TwitterServer
+---
+
+The team working on [Finagle](https://twitter.github.io/finagle/), [Finatra](https://twitter.github.io/finatra/), and related libraries met last week and did our planning for the next three months. These are high-level goals and as such, have varying degrees of confidence and certainty.
+
+#### In the spirit of Spring cleaning:
+
+* Remove Netty 3 support from most protocols. We expect to complete most protocols, however HTTP is a long shot.
+* Upgrade [libthrift](https://thrift.apache.org/) to 0.10.0 from the ancient fork of 0.5.0.
```
Comment (taylorleese) [inline source/blog/2017-03-20-spring-planning.md]: Link to https://github.com/twitter/scrooge/issues/85?

File: source/blog/2017-03-20-spring-planning.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+* Deprecate and then remove modules that are unsupported or exist in a zombie-like state. This includes [ostrich](https://github.com/twitter/ostrich), [finagle-ostrich4](https://github.com/twitter/finagle/tree/develop/finagle-ostrich4), [finagle-kestrel](https://github.com/twitter/finagle/tree/develop/finagle-kestrel), [finagle-stream](https://github.com/twitter/finagle/tree/develop/finagle-stream), [finagle-native](https://github.com/twitter/finagle/tree/develop/finagle-native), [finagle-mdns](https://github.com/twitter/finagle/tree/develop/finagle-mdns), and [util-eval](https://github.com/twitter/util/tree/develop/util-eval).
+* Minimize dependencies on [twitter-commons](https://github.com/twitter/commons) with the eventual goal of removing all of them.
+
+#### For those interested in the shiny and new:
+
+* Get HTTP/2 to production level quality so that it becomes the default HTTP implementation.
+* Continued SSL/TLS work, including STARTTLS support.
```
Comment (taylorleese) [inline source/blog/2017-03-20-spring-planning.md]: STARTTLS => STARTTLS for mux (per @ryanoneill)

File: source/blog/2017-03-20-spring-planning.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+#### For those who like their tools polished:
+
+* Throughput improvements via changes to Futures, defaulting to the Netty 4 native transport, Netty 4 buffer pooling, and more.
+* Switch the default [load balancer](https://twitter.github.io/finagle/guide/Clients.html#load-balancing) to Aperture.
+* Switch the default [failure accrual](https://twitter.github.io/finagle/guide/Clients.html#failure-accrual) policy to success rate instead of consecutive failures.
+* Move TwitterServer to use [slf4j](https://www.slf4j.org/) for logging and begin the process of moving Util and Finagle.
+* A handful of Scrooge changes to make the Scala generated code more composable, fixing [ResponseClassification](https://twitter.github.io/finagle/guide/Clients.html#response-classification) for servers for example.
+
+I realize details here are sparse and certainly not comprehensive. Please ask questions on the mailing list and hopefully we can help clarify if you would like to know more.
```
Comment (taylorleese) [inline source/blog/2017-03-20-spring-planning.md]: > Please feel free to ask questions on the mailing list about anything that is unclear and we’ll help clarify if you would like to know more.

Comment (mosesn) [inline source/blog/2017-03-20-spring-planning.md]: maybe add a link to the mailing list too

**Comment by kevinoliver** [comment]
@taylorleese thanks. pushed an updated version.

**Pull request merged**